tests/datalab/issue_manager/test_data_valuation.py

```diff
@@ -74,3 +74,11 @@ def test_find_issue_wrong_knn_graph(self, issue_manager, embeddings):
                 features=embeddings["embedding"], issue_types={"outlier": {"k": 3}}
             )
             issue_manager.find_issues(k=4)
+
+    def test_get_larger_k_than_knn_graph(self, issue_manager, embeddings):
+        outlier_issue_manager = OutlierIssueManager(datalab=issue_manager.datalab, k=3)
```

Comment:
Nit: Use the `outlier_issue_manager` fixture instead of creating a new `OutlierIssueManager` object.